frontend/src/metabase/visualizations/echarts/cartesian/model/trend-line.ts

```diff
@@ -59,7 +60,7 @@ const getSeriesModelsWithTrends = (
         return null;
       }
 
-      const trendFunction = getTrendLineFunction(insight);
+      const trendFunction = getTrendLineFunction(insight); // add a multiplier here
```

Comment:
Do we still need the comment?

Reply:
I left it for myself and forgot to remove 😄 Thanks for catching